src/arrayMethodSort.js

```diff
@@ -1,11 +1,34 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const compareFunction = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
```

Comment:
I guess you misunderstood this [comment](https://github.com/mate-academy/js_array-method-sort/pull/842#discussion_r990350108)

You still need cases that return `0` or `-1`
But better to format it in another way (without `else` keyword)
just return `-1` at the end of the function outside `if` statement